backend/tests/test_api.py

```diff
@@ -24,32 +24,31 @@ def test_delete_project(neo4j, client):
         CREATE (n:PROJECT)
         SET n.id = 'test_project'
         SET n.taxonomy_name = 'test_taxonomy_name'
-        SET n.branch_name = 'test_branch_name'
+        SET n.branch = 'test_branch'
         SET n.description = 'test_description'
         SET n.status = 'OPEN'
         SET n.project_name = 'p_test_taxonomy_name_test_branch_name'
         SET n.created_at = datetime()
     """
-    result = session.run(create_project)
 
-    # check if the record was created
-    query = """
-        MATCH (n:PROJECT {id: 'test_project'})
-        RETURN n.id, n.taxonomy_name, n.branch_name, n.description, n.status, n.created_at, n.project_name
+    create_project2 = """
+        CREATE (n:PROJECT)
+        SET n.id = 'test_project2'
+        SET n.taxonomy_name = 'test_taxonomy_name2'
+        SET n.branch = 'test_branch2'
+        SET n.description = 'test_description2'
+        SET n.status = 'OPEN'
+        SET n.project_name = 'p_test_taxonomy_name_test_branch_name2'
+        SET n.created_at = datetime()
     """
-    result = session.run(query)
-    record = result.single()
-
-    assert record['n.id'] == 'test_project'
-    assert record['n.taxonomy_name'] == 'test_taxonomy_name'
-    assert record['n.branch_name'] == 'test_branch_name'
-    assert record['n.description'] == 'test_description'
-    assert record['n.status'] == 'OPEN'
-    assert record['n.created_at'] is not None
-    assert record['n.project_name'] == "p_test_taxonomy_name_test_branch_name"
-    # check ends here
-
-    response = client.delete("/test_taxonomy_name/test_branch_name/delete")
+    session.run(create_project)
+    session.run(create_project2)
+
+    response = client.delete("/test_taxonomy_name/test_branch/delete")
```

Comment:
This test case tests the API. 

Do you think I should also test the delete function in `entries.py`?

Reply:
I don't think it's required at this point, since this indirectly tests the database as well. @alexgarel what do you think?

Reply:
I agree with @aadarsh-ram